GovUk.Frontend.Umbraco.Tests/GovUkTypographyTests.cs

```diff
@@ -129,6 +129,37 @@ public void List_classes_are_applied_to_ordered_lists()
             Assert.AreEqual(2, doc.DocumentNode.SelectNodes("//ol[contains(@class,'govuk-list--number')]").Count);
         }
 
+        [Test]
+        public void Style_attribute_is_removed_from_ordered_lists()
```

Comment:
I first thought that the purpose of this test was to test that any non-permitted styles were removed so I got a bit confused.  Would it be possible to change the title to "**All_style attributes_are**_removed_from_ordered_list"? (If I haven't misunderstood the purpose of the test)

Reply:
The _attribute_ is the HTML, ie `style=""`. The _value_ of the style attribute is a semi-colon separated list of CSS _property: value_ pairs.

It is correct to say that the style attribute (singular) is being removed. A test for what you're talking about would be called `All_CSS_properties_are_removed_from_style_attribute_on_ordered_list", but that might imply that the empty attribute itself remains.